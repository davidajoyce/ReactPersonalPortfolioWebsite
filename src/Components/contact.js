import React, { Component } from 'react';
import { Grid, Cell, List, ListItem, ListItemContent } from 'react-mdl';

class Contact extends Component{
  render(){
      return(
        <div className="contact-body">
          <Grid className="contact-grid">
            <Cell col={4}>
              <h2>David Joyce</h2>
              <img
                src="https://media.licdn.com/dms/image/C4D03AQHnehhdUcKSNQ/profile-displayphoto-shrink_200_200/0?e=1563408000&v=beta&t=2bfKMRNJm9GZTTWJVTMmf_lnU7azEECyrzjPUTrJGPo"
                 alt="avatar"
                 style={{height: '250px'}}
                 />
            </Cell>
            <Cell col={8}>
              <h2>Contact Me</h2>
              <hr/>

              <div className="contact-list">
                <List>
                  <ListItem>
                   <ListItemContent style={{fontSize: '16px', fontFamily: 'Anton'}}>
                   <i className="fa fa-envelope" aria-hidden="true"/>
                   davidajoyce141@gmail.com
                   </ListItemContent>
                  </ListItem>

                </List>
              </div>



            </Cell>
          </Grid>
        </div>
      )
    }
}

export default Contact;
