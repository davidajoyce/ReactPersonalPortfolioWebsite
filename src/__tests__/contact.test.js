import React from 'react'
import contact from '../Components/contact'
import { shallow } from 'enzyme';

describe('contact', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<contact />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});