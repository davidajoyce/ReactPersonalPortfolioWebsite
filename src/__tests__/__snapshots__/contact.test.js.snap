// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact renders correctly 1`] = `ShallowWrapper {}`;
